src/__tests__/__snapshots__/documenter.test.ts.snap

```diff
@@ -2757,6 +2757,13 @@ It prevents users from clicking the button.",
       "optional": true,
       "type": "string",
     },
+    Object {
+      "description": "Adds a \`rel\` attribute to the link. By default, the component sets the \`rel\` attribute to \\"noopener noreferrer\\" when \`target\` is \`\\"_blank\\"\`.
```

Comment:
I think we should set the `defaultValue` to "noopener noreferrer". We state it in the description but not in the API table's "Default" column.
(This should be done for the link component in a follow up PR as well).

Reply:
This `defaultValue` is only "default" if `target="_blank"`, otherwise no value will be set. That's why it's only on the description and not on the API "Default" column.

So, if I have a `<Button href="#">`, no "rel" attribute will be rendered. 

I think that's why we didn't do it for the Link as well. What do you think?